src/main/java/me/totalfreedom/totalfreedommod/command/Command_massmurder.java

```diff
@@ -7,7 +7,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.SENIOR_ADMIN, source = SourceType.BOTH)
-@CommandParameters(description = "seth's personal command", usage = "/<command> <players>", aliases = "genocide")
+@CommandParameters(description = "It's like forcekill, but with even more force - Allows you to kill multiple players at once.", usage = "/<command> <players>", aliases = "genocide")
```

Comment:
Please just delete this command. It's grossly inappropriate and frankly shouldn't be on the fucking server or in SCM...

Reply:
done